content/inside-rust/2025-leadership-council-survey.md

```diff
@@ -0,0 +1,34 @@
++++
+path = "inside-rust/9999/12/31/2025-leadership-council-survey"
+title = "2025 Leadership Council Survey"
+authors = ["Leadership Council"]
+
+[extra]
+team = "the Leadership Council team"
+team_url = "https://www.rust-lang.org/governance/teams/leadership-council#team-leadership-council"
++++
+
+One of the responsibilities of the [leadership council](https://www.rust-lang.org/governance/teams/leadership-council),
+formed by [RFC 3392](https://rust-lang.github.io/rfcs/3392-leadership-council.html),
+is to solicit feedback on a yearly basis from the Project on how we are performing
+our duties.
+
+> Each year, the Council must solicit feedback on whether the Council is
+> serving its purpose effectively from all willing and able Project members and
+> openly discuss this feedback in a forum that allows and encourages active
+> participation from all Project members. To do so, the Council and other
+> Project members consult the high-level duties, expectations, and constraints
+> listed in this RFC and any subsequent revisions thereof to determine if the
+> Council is meeting its duties and obligations.
+
+We are runnning a quick annual survey, primarily for Rust project members to
+provide feedback on the Leadership Council. However, even if you are not a
+member, feel free to fill out the survey too -- we welcome input from those
+interacting with the Council in other capacities.
+
+This survey is anonymous. The Council will go through all answers and release a
+summary after the survey is complete, similar to the one published
+[last year](https://blog.rust-lang.org/2024/08/26/council-survey/).
+
+Please fill out the survey [here](TODO). It will be open for 3 weeks, closing
```

Comment:
Let's set the start date for 30th June then, so that we have a clear start date. Once the survey is ready, I will send you the survey link and we can merge it next Monday.